cctp_primitives/src/proving_system/verifier/ceased_sidechain_withdrawal.rs

```diff
@@ -0,0 +1,21 @@
+use crate::proving_system::verifier::{ZendooVerifier, UserInputs};
+use crate::type_mapping::{FieldElement, FIELD_SIZE};
+
+pub struct CSWProofUserInputs<'a> {
+    _amount:         u64,
+    _sc_id:          &'a [u8; 32],
+    _nullifier:      &'a [u8; FIELD_SIZE],
+    _pub_key_hash:   &'a [u8; MC_PK_SIZE],
+    _reedem_script:  &'a [u8],
+    _cert_data_hash: &'a [u8; FIELD_SIZE],
```

Comment:
To be able to withdraw also FTs happened during the "ceasing period", we have also to pass the SCTXCumTreeHash of the ceasing block as a public input. 
No need for redeem_script